### DISCUSSION TITLE
hacked account

### DISCUSSION THREAD

**Discussion by flagtonhilarino** [community/community#134558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Yesterday I downloaded a code here on the platform and when I ran all my accounts on all social networks, they were logged out and had their email and password changed, I would like help to either delete or recover my account, follow the link 

https://github.com/flagton/

**Comment by ebndev** [comment]
Hi @flagtonhilarino,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
